packages/shared/tests/schema/judge/common.test.ts

```diff
@@ -0,0 +1,28 @@
+import { v4 } from 'uuid';
+import { Schema } from '../../../src';
+
+describe('validates a common schema', () => {
```

Comment:
Sentences are used in `it` blocks; this should just describe the thing being tested 
```suggestion
describe('judge put/post common schema', () => {
```